FND/Agreements/Contracts.rdf

```diff
@@ -325,43 +333,40 @@ A unilateral contract differs from a Bilateral Contract, in which the parties ex
 		<rdfs:label>has execution date time stamp</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;DateTimeStamp"/>
-		<skos:definition>the date and time, including time zone, on which a contract has been fully performed by all parties</skos:definition>
-		<fibo-fnd-utl-av:adaptedFrom rdf:resource="https://definitions.uslegal.com/e/executed-contract/"/>
-		<fibo-fnd-utl-av:explanatoryNote>In other words, the execution date time stamp is the date and time on which the terms of the contract have been completely fulfilled.</fibo-fnd-utl-av:explanatoryNote>
+		<skos:definition>indicates the date and time, including time zone, on which a contract has been signed by all the necessary parties</skos:definition>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-agr-ctr;hasGoverningJurisdiction">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-rel-rel;isGovernedBy"/>
 		<rdfs:label>has governing jurisdiction</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-agr-ctr;Contract"/>
 		<rdfs:range rdf:resource="&fibo-fnd-law-jur;Jurisdiction"/>
-		<skos:definition>the jurisdiction governing the contract, as agreed by all parties</skos:definition>
+		<skos:definition>indicates the jurisdiction governing the contract, as agreed by all parties</skos:definition>
 		<skos:editorialNote>As modeled, this relationship combines two slightly different senses in which a Jurisdiction may be named in some Contract: the jurisdiction under whose laws the contract is deemed to be in force, and the jurisdiction under which the parties agree to submit in the event of any dispute resolution. ScopeNote: One thing to tease out is whether &apos;Dispute Resolution&apos; and other forms of &apos;Governing Law&apos; are one and the same thing or not. Dispute Resolution is uncontroversial, the question is whether there are other implications to Governing Law or if it&apos;s the same thing. For instance I may undertake to behave as though I were responsible to a particular authority i.e., a particular set of statutes.</skos:editorialNote>
 		<fibo-fnd-utl-av:explanatoryNote>In a written contract this is generally identified, for example, as Governing Law, namely the jurisdiction in which any disputes arising from the contract are to be resolved.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:ObjectProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-agr-ctr;hasNonBindingTerms">
 		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;hasContractualElement"/>
 		<rdfs:label>has non-binding terms</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-fnd-agr-ctr;Contract"/>
+		<rdfs:domain rdf:resource="&fibo-fnd-agr-agr;Agreement"/>
 		<rdfs:range rdf:resource="&fibo-fnd-agr-ctr;NonBindingTerms"/>
-		<skos:definition>refers to terms that are included in the contract but are not considered binding</skos:definition>
+		<skos:definition>refers to terms that are included in an agreement that are not considered legally binding</skos:definition>
```

Comment:
remove "refers to"

Reply:
What verb would you prefer?

Reply:
It does not need a verb. The definition reads fine by starting "terms that..."

Reply:
We have a rule you don't like - all properties must be verbs.  Not changing that.